src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -2,104 +2,63 @@
 out: Basic-Def.html
 ---
 
-  [More-About-Settings]: More-About-Settings.html
+  [Task-Graph]: Task-Graph.html
   [Bare-Def]: Bare-Def.html
   [Full-Def]: Full-Def.html
   [Running]: Running.html
   [Library-Dependencies]: Library-Dependencies.html
   [Input-Tasks]: ../docs/Input-Tasks.html
 
-.sbt build definition
----------------------
+Build definition
+----------------
 
 This page describes sbt build definitions, including some "theory" and
 the syntax of `build.sbt`. It assumes you know how to [use sbt][Running]
 and have read the previous pages in the Getting Started Guide.
 
-### Three Flavors of Build Definition
-
-There are three flavors of build definition:
-
-1. Multi-project `.sbt` build definition
-2. Bare `.sbt` build definition
-3. `.scala` build definition
-
-This page discusses the newest multi-project `.sbt` build definition, which combines the strength
-of the two older flavors, and is suitable for all cases.
-You might come across the other older flavors when dealing with builds in the wild.
-See [bare .sbt build definition][Bare-Def] and [.scala build definition][Full-Def] (later in Getting Started) for more
-on other flavors.
-
-In addition, a build definition can contain files ending in `.scala`, located in the
-`project/` subdirectory of the base directory to define commonly used functions and values.
+This page discusses the `build.sbt` build definition.
 
 ### What is a Build Definition?
 
 After examining a set of directories and processing build definition files, sbt
-ends up with `Project` definitions.
+ends up with a set of `Project` definitions.
 
 In `build.sbt` you might create a [Project](../api/sbt/Project.html) definition of
 the project located in the current directory like this:
 
 ```scala
 lazy val root = (project in file("."))
+  .settings(
+    name := "Hello",
+    scalaVersion := "$example_scala_version$"
+  )
 ```
 
 Each project is associated with an immutable map (set of key-value pairs) describing the project.
 
 For example, one key is `name` and it maps to a string value, the name of
 your project.
-
-*Build definition files do not affect sbt's map directly.*
-
-Instead, the build definition creates a huge list of objects with type
-`Setting[T]` where `T` is the type of the value in the map. A `Setting`
-describes a *transformation to the map*, such as adding a new key-value
-pair or appending to an existing value. (In the spirit of functional
-programming with immutable data structures and values, a transformation
-returns a new map -- it does not update the old map in-place.)
-
-Here is how you associate the `Setting[String]` for the name of
-the project located in the current directory:
+The key-value pairs are listed under the `settings(...)` method as follows:
```

Comment:
Should we call this `.settings(...)` with a leading `.`? That's the nomenclature I tend to use when it's a method I need to call on something else, rather than a free-floating function already available in scope